tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -120,6 +120,31 @@ export const downloadResources = async (args: downloadResourcesArgs): Promise<Do
   return downloads
 }
 
+export const downloadResourceBatchAction = async (
+  args: downloadResourcesArgs
+): Promise<Download[]> => {
+  const { page, names, folder } = args
+  const downloads = []
+  if (folder) {
+    await clickResource({ page: page, path: folder })
+  }
+
+  for (const name of names) {
```

Comment:
can we please refactor that part out into a dedicated function to select multiple resources